content/reference/admin/private_locations/control_plane_example_ecs/index.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Deploying a Control Plane on ECS"
```

Comment:
Mention Fargate?

Reply:
Would make sense, but might also make the title a bit long (it appears in the navigation menu on the left, which gets a bit crowded if the titles are too long), so not 100% sure... 🤔

Reply:
I think I'll add it anyway